DOCS-6433 edge metrics (#50)
mulesoft/docs-visualizer: modules/ROOT/pages/cloudhub-setup.adoc

* Mule 4.1.x patch releases available on March 22, 2019 or later.
* Mule 4.2.x.

> To see your app, its connections, and the related metrics in Anypoint Visualizer, you must also enable the Monitoring agent after you upgrade as described in the next section.

Reviewer: Never abbreviate Anypoint Monitoring. If this is not Anypoint monitoring, then use common nouns: the monitoring agent.
```suggestion
To see your app, its connections, and the related metrics in Anypoint Visualizer, enable the Anypoint Monitoring agent after you upgrade.
```
GLOBALLY remove references to sections, topics, and so forth in the same deliverable, assuming that our topics are independent and reusable.

[NOTE]

> If you currently have Anypoint Monitoring enabled for applications running on 3.9.x-AM or 4.x-AM, app-level monitoring will continue to work for those apps.

Reviewer: Please clarify what 3.9.x-AM and 4.x-AM are, using the approved product names and versions. Also:
```suggestion
If you currently have Anypoint Monitoring enabled for applications running on 3.9.x-AM or 4.x-AM, app-level monitoring continues to work for those apps.
```

> However, metrics on the connection level will not appear. 

Reviewer: What does "ON" refer to here: "about" or "at"? Clarify by replacement. Also:
```suggestion
However, metrics on the connection level do not appear. 
```
GLOBALLY search for future tense and reword to present, unless absolutely necessary.

Author: ```suggestion
However, metrics at the connection level do not appear. 
```
